signature/src/lib.rs

```diff
@@ -153,8 +153,10 @@
 
 #![no_std]
 #![cfg_attr(docsrs, feature(doc_cfg))]
+
 #![doc(
-    html_logo_url = "https://raw.githubusercontent.com/RustCrypto/meta/master/ferris_signer.png",
```

Comment:
@tarcieri 
Should I keep the `ferris_signer` logo or are you fine with the standard one?

Reply:
Could go either way. A standard logo is fine.